code/game/objects/items/weapons/nt_melee.dm

```diff
@@ -96,19 +136,112 @@
 		icon_state = initial(icon_state)
 	..()
 
+/obj/item/weapon/tool/sword/nt/scourge/apply_hit_effect(mob/living/carbon/human/target, mob/living/user, hit_zone)
+	. = ..()
+	if(ishuman(target))
+		var/mob/living/carbon/human/O = target
+		target.stun_effect_act(stun, agony, hit_zone, src)
+		O.say(pick("OH", "LORD", "MERCY", "SPARE", "ME", "HAVE", "PLEASE"))
+
+/obj/item/weapon/tool/sword/nt/spear
+	name = "NT Pilum"
+	desc = "A saint looking short spear, designed for use with a shield or as a throwing weapon. The spear-tip usually breaks after being thrown at a target, but it can be welded into shape again."
+	icon_state = "nt_spear"
+	item_state = "nt_spear"
+	wielded_icon = "nt_spear_wielded"
+	force = 24
+	force_wielded_multiplier = 1.08
+	var/tipbroken = FALSE
+	w_class = ITEM_SIZE_HUGE
+	slot_flags = SLOT_BACK | SLOT_BELT 
+	throwforce = 75 
```

Comment:
might be way too much damage for a thrown spear.

Reply:
the actual damage is throwforce * 0.6 (don't ask me why), I tested this thouroughly

Reply:
ah, I found a way to fix this

Reply:
well, nevermind. the damage is calculated through throwforce*throw_speed/5, but increasing the throw speed for the sake of clarity also makes the item move faster when thrown.